recorder:
  csv_encoding: utf-8
  storage_path: /home/beagle/telemetry/
  transmit_period: 30
